org.adempiere.base/src/org/adempiere/base/IModelFactory.java

```diff
@@ -46,4 +46,5 @@
 	 *	@return PO for Record or null
 	 */
 	public PO getPO (String tableName, ResultSet rs, String trxName);
+
```

Comment:
would be good to remove this unnecessary white space only changes (same for the file above)